app/containers/ContestPage/messages.js

```diff
@@ -2,7 +2,7 @@ import { defineMessages } from 'react-intl';
 
 export default defineMessages({
   empty: {
-    id: 'app.containers.ContestResultsPage.empty',
+    id: 'app.containers.ContestPage.empty',
```

Comment:
Czemu wyniosłeś to do ContestPage? ContestPage w tej chwili jedyne co robi to przekierowuje do listy problemów.